app/views/grade_entry_forms/_react_grades_table.js.jsx.erb

```diff
@@ -185,6 +185,24 @@
       });
     },
     render: function() {
+      // need to filter for each section
+      var sections = <%= raw @sections.to_json %>;
+      if (sections.length > 0) {
+        var section_filters = [];
+        var all = '<%= I18n.t('browse_submissions.sections.all') %>';
+        section_filters.push({name: all, text: all, func: function(g) { return g; } });
+        for (var i = 0; i < sections.length; i++) {
```

Comment:
This is how it's done for all the other section filters...
Do you want me to use a `map` for all of them?